pkg/ddevapp/templates.go

```diff
@@ -40,7 +41,8 @@ services:
     environment:
       - DEPLOY_NAME=local
       - VIRTUAL_HOST=$DDEV_HOSTNAME
-      - VIRTUAL_PORT=80,{{ .mailhogport }}
+      # HTTP_EXPOSE allows for ports accepting HTTP traffic to be accessible from <site>.ddev.local:<port>
```

Comment:
Could you please show the colon format example in this lovely comment? Each of the comments could have an example.